Source/WebKit/WebProcess/GPU/graphics/RemoteImageBufferProxy.cpp

```diff
@@ -194,17 +197,23 @@ ImageBufferBackend* RemoteImageBufferProxy::ensureBackend() const
 {
     if (m_backend)
         return m_backend.get();
+
     RefPtr connection = this->connection();
-    if (connection) {
-        auto error = connection->waitForAndDispatchImmediately<Messages::RemoteImageBufferProxy::DidCreateBackend>(m_renderingResourceIdentifier);
-        if (error != IPC::Error::NoError) {
-            RELEASE_LOG(RemoteLayerBuffers, "[renderingBackend=%" PRIu64 "] RemoteImageBufferProxy::ensureBackendCreated - waitForAndDispatchImmediately returned error: %" PUBLIC_LOG_STRING,
-                m_remoteRenderingBackendProxy->renderingBackendIdentifier().toUInt64(), IPC::errorAsString(error).characters());
-            didBecomeUnresponsive();
-            return nullptr;
-        }
-    }
-    return m_backend.get();
+    if (!connection)
+        return nullptr;
+
+    auto error = connection->waitForAndDispatchImmediately<Messages::RemoteImageBufferProxy::DidCreateBackend>(m_renderingResourceIdentifier);
+    if (error == IPC::Error::NoError)
+        return m_backend.get();
+
+    RefPtr remoteRenderingBackendProxy = m_remoteRenderingBackendProxy.get();
+    if (UNLIKELY(!remoteRenderingBackendProxy))
+        return nullptr;
+
+    RELEASE_LOG(RemoteLayerBuffers, "[renderingBackend=%" PRIu64 "] RemoteImageBufferProxy::ensureBackendCreated - waitForAndDispatchImmediately returned error: %" PUBLIC_LOG_STRING,
```

Comment:
Might be worth continuing to log these errors, just with a 0 identifier if the backend proxy has gone away.

Reply:
Fixed.